### DISCUSSION TITLE
Help needed - I accidentally uploaded a package with a bug. it was already approved.

### DISCUSSION THREAD

**Discussion by LOCOTA** [microsoft/MicrosoftEdge-Extensions#221] [Q&A]
I accidentally uploaded a package with a bug. it was already approved. 
If I upload another one it will take a couple of days for it to get approved...
What can I do???

**Comment by ManikanthMSFT** [comment]
Hey @LOCOTA, we have responded to your query on #220. To avoid any confusion, we are closing this ticket as a duplicate.

Thank you for your understanding.
